acceptance_tests/features/pages/collection_exercise_details.py

```diff
@@ -64,13 +64,14 @@ def get_collection_exercise_events():
         "mps": browser.find_by_name('mps-date').value,
         "go_live": browser.find_by_name('go-live-date').value,
         "return_by": browser.find_by_name('return-by-date').value,
-        "first_reminder": browser.find_by_name('first-reminder-date').value,
+        "reminder": browser.find_by_name('first-reminder-date').value,
```

Comment:
How come we have both reminder and first_reminder?

Reply:
Lazy duplication in this map to match with the [mandatory events](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/45/files#diff-0e08e30e9d752f3472bf800f9e3df467R9) the collection exercise service API knows about.